modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/components/PopoverIconButton/index.tsx

```diff
@@ -10,37 +10,45 @@
  */
 
 import {ClayButtonWithIcon} from '@clayui/button';
-import ClayPopover from '@clayui/popover';
-import i18n from '../../../../../../../../../common/I18n';
+import ClayPopover, {ALIGN_POSITIONS} from '@clayui/popover';
 
-const PopoverIconButton = ({
+type Writeable<T> = {-readonly [P in keyof T]: T[P]};
+
+type PopoverIconButtonProps = {
+	alignPosition?: Writeable<typeof ALIGN_POSITIONS[number]>;
+	articleAccountSupport?: {text: string; url: string};
+	popoverText?: string;
+};
+
+const PopoverIconButton: React.FC<PopoverIconButtonProps> = ({
 	alignPosition = 'bottom',
-	articleAccountSupportURL,
+	articleAccountSupport,
```

Comment:
@rafaelfs98 you told me this is a generic component, so it shouldn't have this `articleAccountSupport`, right?

Also, it could be removed and this `<a` you are using could be passed together with `popoverText` orrrr replace the popoverText by children, because it can be anything.